uber/templates/macros.html

```diff
@@ -119,111 +119,100 @@
+            <input type="text" name="{{ name_prefix }}address1" class="form-control" placeholder="Address Line 1" value="{{ model.address1 }}" />
         </div>
     </div>
-    <div class="form-group {{ id_prefix }}address_details">
-        <label for="{{ id_prefix }}address2" class="col-sm-3 control-label"></label>
-        <div class="col-sm-6">
-            <input type="text" name="{{ name_prefix }}address2" class="form-control" placeholder="Address Line 2" value="{{ group_or_attendee.address2 }}" />
+    <div class="form-group address_details{{ suffix }}">
+        <div class="col-sm-6 col-sm-offset-3">
+            <input type="text" name="{{ name_prefix }}address2" class="form-control" placeholder="Address Line 2" value="{{ model.address2 }}" />
         </div>
     </div>
-    <div class="form-group {{ id_prefix }}address_details">
-        <label for="{{ id_prefix }}city" class="col-sm-3 control-label">City</label>
+    <div class="form-group address_details{{ suffix }}">
+        <label for="city{{ suffix }}" class="col-sm-3 control-label">City</label>
         <div class="col-sm-6">
-            <input type="text" name="{{ name_prefix }}city" class="form-control" placeholder="City" value="{{ group_or_attendee.city }}" />
+            <input type="text" name="{{ name_prefix }}city" class="form-control" placeholder="City" value="{{ model.city }}" />
         </div>
     </div>
-    <div class="form-group {{ id_prefix }}address_details">
-        <label for="{{ id_prefix }}region" class="col-sm-3 control-label">State/Region</label>
+    <div class="form-group address_details{{ suffix }}">
+        <label for="region{{ suffix }}" class="col-sm-3 control-label">State/Region</label>
         <div class="col-sm-6">
-            <input type="text" name="{{ name_prefix }}region" id="{{ id_prefix }}region" class="form-control" placeholder="State/Province" value="{{ group_or_attendee.region }}" autocomplete="address-level1" />
+            <input type="text" name="{{ name_prefix }}region" id="region{{ suffix }}" class="form-control" placeholder="State/Province" value="{{ model.region }}" autocomplete="address-level1" />
         </div>
     </div>
-    <div class="form-group {{ id_prefix }}address_details">
-        <label for="{{ id_prefix }}zip_code" class="col-sm-3 control-label">ZIP/Postal Code</label>
+    <div class="form-group address_details{{ suffix }}">
+        <label for="zip_code{{ suffix }}" class="col-sm-3 control-label">ZIP/Postal Code</label>
         <div class="col-sm-6">
-            <input type="text" name="{{ name_prefix }}zip_code" class="form-control" value="{{ group_or_attendee.zip_code }}"/>
+            <input type="text" name="{{ name_prefix }}zip_code" class="form-control" value="{{ model.zip_code }}"/>
         </div>
     </div>
 {%- endmacro %}
+
+
+{% macro toggle_checkbox(selector, label, suffix='', name='', value='1', checked=False) -%}
```

Comment:
Nice! If we want to, in the future we could use this for the same_legal_name checkbox too. Right now that disables the field but hiding it could be just as good.